manifests/a/AVerMedia/AssistCentral/1.2.0.13/AVerMedia.AssistCentral.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AVerMedia.AssistCentral
+PackageVersion: 1.2.0.13
+PackageLocale: en-US
+Publisher: AVerMedia TECHNOLOGIES, Inc.
+PackageName: AVerMedia Assist Central
+License: © 2022 AVerMedia Technologies, Inc.

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate